bom/aissemble-quarkus-bom/pom.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.boozallen.aissemble</groupId>
+        <artifactId>bom</artifactId>
+        <version>1.10.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>aissemble-quarkus-bom</artifactId>
+    <packaging>pom</packaging>
+
+    <name>aiSSEMBLE::BOM::Quarkus</name>
+    <description>An aiSSEMBLE compabible version of the quarkus-universe-bom</description>
+
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-universe-bom</artifactId>
```

Comment:
I: Moving quarkus-universe-bom to a separate BOM so non quarkus apps can use the standard BOM without bringing in unnecessary quarkus dependencies.